### ISSUE TITLE
Please don't usa N/A

### ISSUE THREAD

**Issue by BryanQuigley** [CanonicalLtd/ubuntu-advantage-client#476]
My default run on trusty.  If we don't like this suggestion, let's at least spell out not available for release, so it's clear why it's n/a (and what n/a means)
....

cc            entitled         n/a
cis-audit     none             n/a
esm           entitled         active
fips          entitled         n/a
fips-updates  none             n/a
livepatch     entitled         n/a

Instead I think it should only show items available on a release something like:

...
14.04 LTS entitlements:
esm           active

Then --verbose can show

16.04 LTS entitlements:
cc       
cis-audit         
fips    
fips-updates
livepatch 

18.04 LTS enttilements
livepatch 
fips - coming soon!

Somewhat related to #419

**Comment by cpaelzer** [comment]
I think (opinion) it is fine to show the `n/a` as to some extend it will make users available that there is more.
But I agree that the (not yet existin) --verbose should add more detail why things are missing.

My mockup would more look like:
```
ubuntu@trusty-ua:~$ sudo ua status
Account: christian.ehrhardt@canonical.com
Subscription: christian.ehrhardt@canonical.com
Valid until: n/a
Technical support level: n/a

cc            entitled         n/a    *1
cis-audit     none             n/a    *1
esm           entitled         active
fips          entitled         n/a    *2
fips-updates  none             n/a    *2
livepatch     entitled         n/a    *3

*1): not available in containers
*2): only available on Ubuntu 16.04 and later
*3): only available with Ubuntu kernels
```

Essentially your rules checking if features are applicable could add "hints" for each check done.
That way there also is a chance that some get multiple mentions like
```
livepatch     entitled         n/a    *1 *3

*1): not available in containers
*2): only available on Ubuntu 16.04 and later
```

**Comment by OddBloke** [comment]
Hey Bryan, thanks for trying out the client and filing this issue. The "n/a" is driven by the product folks, so I don't think we're going to change that (or, rather, I don't think having an open GitHub issue is going to drive us changing that). As such, I'm going to close this out.

(@cpaelzer We have #419 to capture getting more information from the `ua status` command.)
